src/gatsby-browser.js

```diff
@@ -1,5 +1,15 @@
-export function onRouteUpdate({ location }) {
-    if (process.env.NODE_ENV !== 'production' || !Array.isArray(window._hsq)) {
+import defaultOptions from './default-options';
+
+export function onRouteUpdate({ location }, pluginOptions) {
+    const { productionOnly } = Object.assign(
+        defaultOptions,
+        pluginOptions
+    );
```

Comment:
This needs to be changed to the following:

```jsx
    const { productionOnly } = Object.assign(
		{},
        defaultOptions,
        pluginOptions
    );
```

Adding an extra empty object as the first parameter to `Object.assign` prevents `defaultOptions` from being mutated. We also need to make this change in `gatsby-ssr.js` if you'd like to update it in the same PR!